ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/GODictionaryRenderer/GODictionaryRenderer.cpp

```diff
@@ -90,17 +91,34 @@ void GODictionaryRenderer::componentComplete()
   const QString specType = QStringLiteral("mil2525d");
   const QString styleLocation = m_dataPath + "/styles/arcade_style/mil2525d.stylx";
   DictionarySymbolStyle* dictionarySymbolStyle = DictionarySymbolStyle::createFromFile(styleLocation, this);
-  DictionaryRenderer* renderer = new DictionaryRenderer(dictionarySymbolStyle, this);
-  m_graphicsOverlay->setRenderer(renderer);
-  //! [Apply Dictionary Renderer Graphics Overlay Cpp]
 
+  // The style will be loaded automatically
+  connect(dictionarySymbolStyle, &DictionarySymbolStyle::loadStatusChanged, this, [dictionarySymbolStyle]
+  {
+    if (dictionarySymbolStyle->loadStatus() == LoadStatus::Loaded)
+    {
+      const auto dictionarySymbolStyleConfigurations = dictionarySymbolStyle->configurations();
```

Comment:
I don't mind using auto here but I noticed in the past any time I used auto I received the feedback that we should be using explicit types. Should this be changed to specifically include the type?